packages/protocol/script/upgrade/UpgradeScript.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.18;
+
+import "forge-std/Script.sol";
+import "forge-std/console2.sol";
+import
+    "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts-upgradeable/utils/math/SafeCastUpgradeable.sol";
+
+contract UpgradeScript is Script {
+    uint256 public deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+
+    address public proxyAddress = vm.envAddress("PROXY_ADDRESS");
+
+    TransparentUpgradeableProxy proxy;
+
+    function setUp() internal {
```

Comment:
Shall we make this a modifier and add:
```solidity
_;
vm.stopBroadcast();
```
at the end?

Reply:
good idea, done!